src/pages/index.js

```diff
@@ -28,35 +28,47 @@ const api = new Api({
   }
 });
 
-Promise.all([api.getUserInfo()])
-  .then(([userData]) => {
+const cardSelector=selectors.cardTemplate;
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, cardData]) => {
     userInfo.setUserInfo(userData);
+    userId = userData._id;
+    newCardSection = new Section(
+      {
+        items: cardData,
+        renderer: (data) => {
+        const newCard = renderCard(data);
+        newCardSection.addItem(newCard);
+          },
+      },
+      selectors.cardSection
+    );
+    newCardSection.renderItems();
 })
-  //.then(res => res.json())
-  //.then((result) => {
-    //console.log(result);
- // })
   .catch(console.error);
 
-const section = new Section({
-  items: initializeCards,
-  renderer: (cardData) => {
-    const cardElement = renderCard(cardData);
-    section.addItem(cardElement);
-    },
-  },
-  cardsList,
-);
-
-section.renderItems();
-
 //edit form
 const userInfo = new UserInfo({
   profileDescriptionSelector: ".profile__description",
   profileNameSelector: ".profile__title",
   profileAvatarSelector: ".profile__img",
 })
 
+api.clearAllCards()
```

Comment:
Needs correction: please remove or comment this, you receive the cards on line 33